codecov.yml

```diff
@@ -1,2 +1,11 @@
 ignore:
   - "test" # exclude 'test' directory from coverage
+  - "core/sector_storage/stores/impl/remote_*"
+  - "core/sector_storage/impl/remote_*"
+  - "core/sector_storage/impl/fetch_handler.cpp"
+  - "core/sector_storage/zerocomm/*"
```

Comment:
I think only `zerocomm` should be ignored.
We should ignore files in coverage not because we want to increase metrics, but because we do not want to test them. `zerocomm` contains wrappers of side project, but the rest is our code and I don't see reason why we should ignore them.

Reply:
Agreed